gym/wrappers/gray_scale_observation.py

```diff
@@ -1,17 +1,32 @@
+"""Wrapper that converts a colour observation to greyscale."""
```

Comment:
Because the repo is mostly American English:
```suggestion
"""Wrapper that converts a color observation to grayscale."""
```